<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTRtmpStreamerClass</class>
 <widget class="QWidget" name="QTRtmpStreamerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTRtmpStreamer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
QWidget
{
	background-color: rgb(53, 53, 53);
}

QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }

QLineEdit
{
border-radius:10px;
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;

}
QComboBox
{
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;

}</string>
  </property>
  <widget class="QLineEdit" name="outUrl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>rtmp://192.168.10.181/live</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RTMP推流 URL ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入的 URL/0/1 ：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inUrl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开始推流</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>美颜级别 ：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="facelevel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三级</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTRtmpStreamer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>QTRtmpStreamerClass</receiver>
   <slot>Stream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Stream()</slot>
 </slots>
</ui>
